src/test/end-to-end/tests/system-admin/bulk-create-media-accounts-test.ts

```diff
@@ -13,7 +36,8 @@ Scenario('I as a system admin should be able to bulk create media accounts', asy
     I.see(
         'Note the upload process has a maximum of 30 accounts created per run. Please ensure the file uploaded for processing has no more than 30 cases.'
     );
-    I.attachFile('#bulk-account-upload', './shared/mocks/bulkCreateUserValid.csv');
+    I.wait(10);
```

Comment:
Are the waits still needed here? If so I wonder if we can do this with something like I wait for text, rather than waiting a specific time

Reply:
No, just forgot to remove that.  I will remove it:)